config/checkstyle/checkstyle.xml

```diff
@@ -108,7 +108,7 @@
             <property name="maxPrivate" value="35"/>
             <property name="maxPackage" value="30"/>
             <property name="maxProtected" value="10"/>
-            <property name="maxPublic" value="13"/>
+            <property name="maxPublic" value="15"/>
```

Comment:
I think you discussed this with @Tim-Hammonds already but I'd be keen to hear the rationale behind this